notNeededPackages.json

```diff
@@ -2756,11 +2761,6 @@
             "typingsPackageName": "multicodec",
             "asOfVersion": "2.1.0"
         },
-        {
-            "libraryName": "multiaddr",
-            "typingsPackageName": "multiaddr",
-            "asOfVersion": "7.4.0"
-        },
```

Comment:
This was moved by the `not-needed` script.  I left in in for this PR so it's not something that happened for everybody in the future, but happy to exclude from my branch if you prefer to keep the changes minimal.